Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM node:alpine
+FROM node:10
```

Comment:
NodeJS v10 is pretty old and not supported anymore. Are there any reasons we can't use the latest LTS version as it was before?

Reply:
Thats my bad, mcrypt needs the full version of node and not specially v10.